x-pack/legacy/plugins/siem/public/containers/timeline/all/index.tsx

```diff
@@ -91,9 +92,10 @@ const AllTimelinesQueryComponent: React.FC<OwnProps> = ({
       notifyOnNetworkStatusChange
       variables={variables}
     >
-      {({ data, loading }) =>
+      {({ data, loading, refetch }) =>
         children!({
           loading,
+          refetch,
```

Comment:
please follow the same concept than here -> x-pack/legacy/plugins/siem/public/containers/timeline/index.tsx

Reply:
Are you mentioning about the inspect button?

Reply:
After the zoom chat, agreed to wrap refetch with useCallback first before passing it down.